.gitignore

```diff
@@ -223,5 +223,9 @@ numpy/core/src/umath/loops_minmax.dispatch.c
 numpy/core/src/umath/loops_trigonometric.dispatch.c
 numpy/core/src/umath/loops_exponent_log.dispatch.c
 numpy/core/src/umath/loops_umath_fp.dispatch.c
+numpy/core/src/umath/loops_hyperbolic.dispatch.c
 # npysort module
 numpy/core/src/npysort/x86-qsort.dispatch.c
+# multiarray module
+numpy/core/src/multiarray/argfunc.dispatch.c
+numpy/core/src/multiarray/arraytypes.h
```

Comment:
`arraytypes.h` isn't a generated file.

Reply:
It seems to be generated from [`arraytypes.h.src`](https://github.com/numpy/numpy/blob/main/numpy/core/src/multiarray/arraytypes.h.src) ever since https://github.com/numpy/numpy/pull/20846.